### PR TITLE
Pin to minor versions instead of latest

### PR DESCRIPTION
**Pull request by marccampbell**
This needs to be verified on vendor web staging after merge

### CODE REVIEW

File: web/src/installers/index.ts [hunk 1]
```diff
@@ -762,15 +762,15 @@ export class Installer {
...
-    i.spec.ekco = { version: "latest" };
-    i.spec.contour = { version: "latest" };
-    i.spec.registry = { version: "latest" };
-    i.spec.prometheus = { version: "latest" };
+    i.spec.kubernetes = { version: "1.21.x" };
+    i.spec.containerd = { version: "1.4.x" };
+    i.spec.weave = { version: "2.6.x" };
+    i.spec.longhorn = { version: "1.1.x"};
+    i.spec.minio = { version: "2020-01-25T02-50-51Z"};
+    i.spec.ekco = { version: "0.12.x" };
```
Comment (emosbaugh) [inline web/src/installers/index.ts:770]: should this be latest? ekco version updates usually have important fixes for better operation of the cluster.

**Review by DanStough — APPROVED** [review]

File: web/src/installers/index.ts (hunk 1 above)
Comment (marccampbell) [reply web/src/installers/index.ts:770]: @emosbaugh sure. it's weird to be the only one...  we don't support 0.x notation yet?  what options can we use to allow us to introduce a breaking change in the future?

Comment (emosbaugh) [reply web/src/installers/index.ts:770]: pinning to major notation eg 0.x would be great. additionally there have been discussions about making ekco built into kurl and no longer an addon.

Comment (emosbaugh) [inline web/src/installers/index.ts:770]:
```suggestion
    i.spec.ekco = { version: "latest" };
```

**Review by emosbaugh — CHANGES_REQUESTED** [review]

**Comment by emosbaugh** [comment]
[sc-38729]

**Review by emosbaugh — APPROVED** [review]

File: web/src/installers/index.ts [hunk 2]
```diff
@@ -762,15 +762,15 @@ export class Installer {
...
+    i.spec.kubernetes = { version: "1.21.x" };
+    i.spec.containerd = { version: "1.4.x" };
+    i.spec.weave = { version: "2.6.x" };
+    i.spec.longhorn = { version: "1.1.x"};
+    i.spec.minio = { version: "2020-01-25T02-50-51Z"};
     i.spec.ekco = { version: "latest" };
-    i.spec.contour = { version: "latest" };
-    i.spec.registry = { version: "latest" };
-    i.spec.prometheus = { version: "latest" };
+    i.spec.contour = { version: "1.18.x" };
```
Comment (DanStough) [inline web/src/installers/index.ts:771]: contour is 1.19 now.

File: web/src/installers/index.ts [hunk 3]
```diff
@@ -762,15 +762,15 @@ export class Installer {
...
     i.id = "latest";
-    i.spec.kubernetes = { version: "latest" };
-    i.spec.containerd = { version: "latest" };
-    i.spec.weave = { version: "latest" };
-    i.spec.longhorn = { version: "latest"};
-    i.spec.minio = { version: "latest"};
+    i.spec.kubernetes = { version: "1.21.x" };
+    i.spec.containerd = { version: "1.4.x" };
+    i.spec.weave = { version: "2.6.x" };
+    i.spec.longhorn = { version: "1.1.x"};
```
Comment (DanStough) [inline web/src/installers/index.ts:768]: Longhorn is 1.2

**Review by DanStough — APPROVED** [review]

**Review by DanStough — DISMISSED** [review]

**Pull request merged**